tests/jest/views/ProjectsTab.spec.js

```diff
@@ -281,6 +282,19 @@ describe('ProjectsTab.vue Test', () => {
 			expect(workPackages).toMatchSnapshot()
 		})
 	})
+
+	describe('action menu', () => {
+		it('should display the action menu in linked work packages', async () => {
+			wrapper = mountWrapper()
+			await wrapper.setData({
+				workpackages: workPackagesSearchResponse,
+			})
+			const toggleMenu = wrapper.find(toggleMenuSelector)
```

Comment:
```suggestion
			const toggleMenu = wrapper.find(toggleMenuSelector)
			expect(toggleMenu.exists()).toBeFalsy()
```
just to verify that the menu exists after the click.